### PR TITLE
Fix linux instructions

### PR DESCRIPTION
**Pull request by elucidsoft**
Linux instructions were incorrect.

### CODE REVIEW

File: nats-server/installation.md [hunk 1]
```diff
@@ -74,12 +74,12 @@ Download the zip file matching your systems architecture, and unzip. For this ex
...
-  inflating: nats-server-v2.0.0-darwin-amd64/README.md
-  inflating: nats-server-v2.0.0-darwin-amd64/LICENSE
-  inflating: nats-server-v2.0.0darwin-amd64/nats-server
+   creating: nats-server-v2.0.0-linux-amd64/
+  inflating: nats-server-v2.0.0-linux-amd64/README.md
+  inflating: nats-server-v2.0.0-linux-amd64/LICENSE
+  inflating: nats-server-v2.0.0-linux-amd64/nats-server
 
-> cp nats-server-v2.0.0darwin-amd64/nats-server /usr/local/bin
+> sudo cp nats-server-v2.0.0-linux-amd64/nats-server /usr/bin
```
Comment (kozlovic) [inline nats-server/installation.md]: I get the `-linux-` change, but I think we should keep `/usr/local/bin`, no?

Comment (gcolliso) [reply nats-server/installation.md]: If that is correct then yes - does `local` change if you are using `sudo`?

Comment (gcolliso) [reply nats-server/installation.md]: @elucidsoft thoughts on the comments above?

Comment (elucidsoft) [reply nats-server/installation.md]: local is not as common as /usr/bin for Linux distributions.

Comment (kozlovic) [reply nats-server/installation.md]: I believe that user executables should preferably be copied to `/usr/local/bin` and OS executables are usually the ones in `/usr/bin`. But I am fine with the change. Thanks!

**Pull request merged**